patches/tModLoader/Terraria/ModLoader/ModMenu.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using ReLogic.Content;
+using Terraria.GameContent;
+using Terraria.UI;
+
+namespace Terraria.ModLoader
+{
+	/// <summary>
+	/// A class that is used to customize aesthetic features of the main menu, such as the logo, background and music.
+	/// </summary>
+	public abstract class ModMenu : ModType
+	{
+		internal UserInterface userInterface;
+
+		internal bool isNew;
+
+		protected sealed override void Register() {
+			MenuLoader.Add(this);
+			ContentInstance.Register(this);
+		}
+
+		internal void SelectionInit() {
+			userInterface = new UserInterface();
+			OnSelected(userInterface);
+		}
+
+		/// <summary>
+		/// The logo texture shown when this ModMenu is active. If not overridden, it will use the tModLoader logo.
+		/// </summary>
+		public virtual Asset<Texture2D> Logo => ModLoader.ManifestAssets.Request<Texture2D>("Terraria.ModLoader.Logo");
```

Comment:
Cache this pls and ty

Reply:
I copied what the drawcode used in Main, so I'm not sure of any alternative